pkg/network/tracer/tracer.go

```diff
@@ -57,6 +62,8 @@ type Tracer struct {
 	ebpfTracer   connection.Tracer
 	bpfTelemetry *telemetry.EBPFTelemetry
 
+	clientConnections map[string]connections
```

Comment:
This could use a comment explaining how it's used.  It took some reading to figure out that this field was used to cache the last run of the tracer

Reply:
done